ui/raidboss/data/04-sb/raid/o3s.txt

```diff
@@ -1,176 +1,136 @@
+28.4 "Holy Blur/Holy Edge" sync /:Halicarnassus:22F[01]:/
+37.4 "the Queen's Waltz (Clock)" sync /:Halicarnassus:2306:/ window 37.4,5
+37.9 "Sword Dance" sync /:Halicarnassus:2307:/
+41.4 "Haste" sync /:Halicarnassus:22F4:/
+44.8 "Spellblade Thunder III" sync /:Halicarnassus:22EE:/
+50.3 "Dimensional Wave" sync /:Halicarnassus:22F6:/
+59.3 "the Playing Field" sync /:Halicarnassus:2300:/
+65.3 "Oink" sync /:Halicarnassus:22F9:/
+72.3 "the Game" sync /:Halicarnassus:2301:/ window 72.3,10
 
-########Phase 2
-81	"Panel Swap"	sync /:Halicarnassus:2304:/ window 30	#Arena changes
-90	"Queen's Waltz (Tethers)"	#Applies thorny tethers
-93	"Tethers"	#Run from partner Healer/Tank always tethered to a DPS. Never Healer-Healer, tank-tank, tank-healer, dps-dps.
-95	"Fire"	#Spellblade Fire III
-99	"Haste"
-103	"Thunder"	#Spellblade Thunder III
-108	"Critical Hit"	#Tank Buster
-115	"Dimensional Wave"	#AoE
-121	"Place Token"	#Spawns add
-122	"--White Flame Spawns--"
-132	"Haste"
-136	"Blizzard"	#Spellblade Blizzard III
-143	"Mindjack" duration 13  #Applies 13s facing debuffs
-155	"Queen's Waltz (Tethers)"	#Applies thorny tethers
-157	"Tethers"	#Run from partner
-160	"Fire"	#Spellblade Fire III
-167	"Dimensional Wave"	#AoE
+81.2 "Panel Swap (Thorns)" sync /:Halicarnassus:2304:/
+83.2 "--center--" sync /:Halicarnassus:2305:/
+89.5 "the Queen's Waltz (Tethers)" sync /:Halicarnassus:2308:/
+94.2 "Spellblade Fire III" sync /:Halicarnassus:22EC:/
+98.2 "Haste" sync /:Halicarnassus:22F4:/
+101.5 "Spellblade Thunder III" sync /:Halicarnassus:22EE:/
+107.0 "Critical Hit" sync /:Halicarnassus:22EB:/
+113.0 "Dimensional Wave" sync /:Halicarnassus:22F6:/ window 30,30
+119.0 "Place Token (White Flame)" sync /:Halicarnassus:22FB:/
+124.9 "Ray of White" # sync /:White Flame:2310:/
+129.9 "White Wind" # sync /:White Flame:2311:/
+130.0 "Haste" sync /:Halicarnassus:22F4:/
+131.9 "Ray of White" # sync /:White Flame:2310:/
+133.5 "Spellblade Blizzard III" sync /:Halicarnassus:22ED:/
+135.9 "Ray of White" # sync /:White Flame:2310:/
+140.5 "Mindjack" sync /:Halicarnassus:22FA:/ window 140,10
+146.0 "--center--" sync /:Halicarnassus:2305:/
+152.6 "the Queen's Waltz (Tethers)" sync /:Halicarnassus:2308:/
+157.6 "Spellblade Fire III" sync /:Halicarnassus:22EC:/
+163.6 "Dimensional Wave" sync /:Halicarnassus:22F6:/
 
-########Phase 3
-174	"Panel Swap"	sync /:Halicarnassus:2304:/ window 30	#Arena changes
-180	"Place Token"	#spawns add
-181	"--Great Dragon Spawns--"	#Spawns add
-191	"Queen's Waltz (Crystals)"	#AoE + Stand on blue tiles
-197	"Ribbit"
-203	"Dimensional Wave"	#Dimensional Wave
-203	"Dragon Conal AoE" #Frostbite
-217	"Critical Hit"	#Tank Buster
-228	"Place Dark Token"	#spawns 5 untargetable adds
-240	"Queen's Waltz (Crystals)"	#AoE + Stand on blue tiles
-#242	"Cross Reaper/Gusting Gouge"	#Middle AoE + knockback on edge tiles
-248	"Random Elemental" #Fire, Blizz, or Thunder
-253	"Dimensional Wave"	#AoE
+171.1 "Panel Swap (Cave)" sync /:Halicarnassus:2304:/ window 85,10
```

Comment:
If this is a push, can you leave a comment (and maybe sync all the way back to the beginning?)

Is it worth putting larger syncs on other panel swap phases? (I could go either way, honest question.)

Reply:
I don't know for sure. I'm mostly trying to ensure that we have the widest possible safe sync windows for those situations where we forget to turn ACT on until halfway through. (Also unfortunately we can't sync either The Game or Panel Swap to the beginning since it's the same ability each time.) If that's not worthwhile I can cut them down to the normal 30,30 windows.

Reply:
Oh, ok.  So, not a push, just an extra sync.  Sounds good!  I think this is fine.  :)